moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -206,30 +213,43 @@ const ProjectFundingTable = () => {
   const getLookupValueByID = (lookupTable, attribute, id) => {
     if (!id) return null;
 
-    return data[lookupTable].find(item => item[`${attribute}_id`] === id)[
+    return data[lookupTable].find((item) => item[`${attribute}_id`] === id)[
       `${attribute}_name`
     ];
   };
 
+  const userId = getDatabaseId(user);
+
   /**
    * An array of objects that contain the task order data
    * @type {Array[Object]}
    */
   const taskOrderData = data?.moped_project?.[0]?.task_order ?? [];
 
+  /**
+   * concatanates fund dept and unit ids into an fdu string
+   * @type {Array[Strings]}
+   */
+  const fdusArray = data?.moped_proj_funding.map(
+    (record) =>
+      `${record.fund?.fund_id} ${record.dept_unit?.dept} ${record.dept_unit?.unit}`
+  );
```

Comment:
i think we're talkin microseconds, but here is a place where you could `useMemo()` to only do this munging once when get the data.